app/views/miq_request/_request_dialog_details.html.haml

```diff
@@ -0,0 +1,36 @@
+.form-group{:id => "field_#{field.id}_tr", :style => field.visible ? 'display:block' : 'display:none'}
+  %label.control-label.col-md-2{:title => field.description}
+    = field.label
+  .col-md-8{:title => field.description}
+    - case field.type
+    - when 'DialogFieldTextBox', 'DialogFieldTextAreaBox'
+      - if field.protected?
+        ********
+      - else
+        = h(field.value)
+
+    - when 'DialogFieldCheckBox'
+      = check_box_tag(field.name, "1", field.checked?, {:disabled => !edit})
+
+    - when 'DialogFieldDateControl'
+      = h(field.value)
+
+    - when 'DialogFieldDateTimeControl'
+      - date_val, time_val = field.value.split(" ")
+      - hour_val, minute_val = time_val.split(":")
+      = date_val
+      &nbsp;at&nbsp;
+      = "#{hour_val.rjust(2, '0')}:#{minute_val.rjust(2, '0')}"
+      &nbsp;
+      = session[:user_tz]
+
+    - when "DialogFieldRadioButton"
+      = h(field.values.detect { |k, _v| k == wf.value(field.name) }.try(:last) || wf.value(field.name))
```

Comment:
@eclarizio same comment for `wf` variable here and on line 34

Reply:
Good point, will pass it into the locals in `_st_prov_show`